src/collective/easyform/fields.py

```diff
@@ -26,15 +25,36 @@
 from zope.schema.interfaces import IField
 
 
+@implementer(IForm)
+class GenericFormWrapper(object):
+    def __init__(self, view):
+        self.__view__ = view
+
+    def __getattr__(self, item):
+        return getattr(self.__view__, item)
+
+
 @implementer(IValidator)
 @adapter(IEasyForm, Interface, IEasyFormForm, IField, Interface)
-class FieldExtenderValidator(z3c_validator.SimpleFieldValidator):
-
+class FieldExtenderValidator(object):
     """ z3c.form validator class for easyform fields in the default fieldset"""
 
+    def __init__(self, context, request, view, field, widget):
+        self.context = context
+        self.request = request
+        self.view = view
+        self.field = field
+        self.widget = widget
+
     def validate(self, value):
         """ Validate field by TValidator """
-        super(FieldExtenderValidator, self).validate(value)
+        # First see if there is a another validator adapter for the field which isn't us
+        view = GenericFormWrapper(self.view)
```

Comment:
This could use a comment about what does does and why it works.
Is this to make a fieldset and form behave the same?

Reply:
The original problem is that this validator got in the way of the recaptcha validator so it never got called as the registration was more specifc. Instead of just registering the recaptcha validator I figured this was a more general solution as there could be other validator adapters being blocked in the future.